src/js/structs/__tests__/Deployment-test.js

```diff
@@ -0,0 +1,16 @@
+let Deployment = require('../Deployment');
```

Comment:
Sure. For my own interest, could you give me a quick run-down on why that would be a useful test?